Campaign.html

```diff
@@ -1,188 +0,0 @@
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-<title>Vannessa Phillips •  Visual &amp; Web Design</title>
-<!--
-/*
- * MyFonts Webfont Build ID 755816, 2011-04-12T16:02:46-0400
- * 
- * The fonts listed in this notice are subject to the End User License
- * Agreement(s) entered into by the website owner. All other parties are 
- * explicitly restricted from using the Licensed Webfonts(s).
- * 
- * You may obtain a valid license at the URLs below.
- * 
- * Webfont: Still Time
- * URL: http://new.myfonts.com/fonts/larabie/still-time/still-time/
- * Foundry: Larabie
- * Copyright: (c) 1999 Ray Larabie. See attached license agreement for more information. If EULA is missing, visit www.larabiefonts.com for an updated version of this font.
- * License: http://www.myfonts.com/viewlicense?1056
- * Licensed pageviews: unlimited/month
- * CSS font-family: StillTime-Regular
- * CSS font-weight: normal
- * 
- * Webfont: Otari-Bold-Limited
- * URL: http://new.myfonts.com/fonts/tk-type/otari/bold-limited/
- * Foundry: TK Type
- * Copyright: Copyright (c) 2010 by Travis Kochel. All rights reserved.
- * License: http://www.myfonts.com/viewlicense?1056
- * Licensed pageviews: unlimited/month
- * CSS font-family: Otari-Bold-Limited
- * CSS font-weight: normal
- * 
- * Webfont: District Book
- * URL: http://new.myfonts.com/fonts/garagefonts/district/book/
- * Foundry: GarageFonts
- * Copyright: Copyright (c) 2002 by Dylan Smith &amp; Kienan Smith. Published by GarageFonts USA 301-879-9222 - - www.garagefonts.com. All rights reserved.
- * License: http://www.myfonts.com/viewlicense?1056
- * Licensed pageviews: 10,000/month
- * CSS font-family: District-Book
- * CSS font-weight: normal
- * 
- * (c) 2011 Bitstream, Inc
-*/
-
-
--->
-<script type="text/javascript">
-//change this to true to enable troubleshooting mode
-var myfont_webfont_test = false;
-// change this to false if you're having trouble with WOFFs
-var woffEnabled = true;
-// to place your webfonts in a custom directory, uncomment this and set it to where your webfonts are.
-//var customPath = "/themes/fonts";
-</script>
-<script type="text/javascript" src="MyFontsWebfontsOrderM2858474.js"></script>
-<link href="style.css" rel="stylesheet" type="text/css" />
-
-    <script type="text/javascript">
-
-  var _gaq = _gaq || [];
-  _gaq.push(['_setAccount', 'UA-21871866-1']);
-  _gaq.push(['_trackPageview']);
-
-  (function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-  })();
-
-</script>
-  
-</head>
-   
-         
-         
-  
-<body>
-<div class="wrapper">
-<div id="jar"><br />
```

Comment:
be careful with your indentation - in general, things inside block elements (like a div) should be indented once - and opening and closing tags should always line up with each other, unless they're on the same line